app/views/projects/show.html.erb

```diff
@@ -3,7 +3,7 @@
   <h1><%=t '.title', :default => model_class.model_name.human.titleize %></h1>
 </div>
 <div class="row show-container">
-  <div class="offset-4 col-4 ">
+  <div class="col-xs-12 col-xs-offset-0 col-md-8 col-md-offset-2">
```

Comment:
Bootstrap will assume `col-xs-12` and `col-xs-offset-0` if you only specify `col-md-*`, so you don't need to do that. Pretty cool!